receiver/collectdreceiver/go.sum

```diff
@@ -467,6 +467,7 @@ github.com/onsi/gomega v1.5.0 h1:izbySO9zDPmjJ8rDjLvkA2zJHIo+HkYXHnf7eN7SSyo=
 github.com/onsi/gomega v1.5.0/go.mod h1:ex+gbHU/CVuBBDIJjb2X0qEXbFg53c61hWP/1CpauHY=
 github.com/onsi/gomega v1.7.0 h1:XPnZz8VVBHjVsy1vzJmRwIcSwiUO+JFfrv/xGiigmME=
 github.com/onsi/gomega v1.7.0/go.mod h1:ex+gbHU/CVuBBDIJjb2X0qEXbFg53c61hWP/1CpauHY=
+github.com/open-telemetry/opentelemetry-collector v0.2.7 h1:8zWYD+iIkvvV3B0rElgUUmpgujqZyIDQKLOw3lpotzI=
```

Comment:
interesting that this showed up here, perhaps something triggered by your IDE?

Reply:
These were added when doing the build with make.